R/pmcmc.R

```diff
@@ -362,9 +361,9 @@ pmcmc <- function(data,
 }
 
 
-calc_loglikelihood <- function(pars, data, sircovid_model, model_params, model_start_date,
+calc_loglikelihood <- function(pars, data, sircovid_model, model_params,
                                steps_per_day, pars_obs, n_particles) {
-  start_date <- as.Date(pars[['start_date']], origin=model_start_date)
+  start_date <- as.Date(-pars[['start_date']], origin=data$date[1])
```

Comment:
This is the key change, looks like it makes sense in the debugger.

Can you just add a quick comment to explain that start_date is an int which we subtract from the first observed date in the data? Then good to merge